test/integration/features/cmd-openshift.feature

```diff
@@ -35,13 +41,21 @@ cluster in VM provided by Minishift.
       Then stdout should contain "OpenShift restarted successfully"
        And stdout of command "minishift ssh -- "docker inspect --format={{.State.FinishedAt}} origin"" is not equal to "0001-01-01T00:00:00Z"
 
+  Scenario: registry check
+     Given executing "oc login -u system:admin" succeeds
+      When executing "oc project default" succeeds
+      When executing "oc get -o json pods" succeeds
+      Then stdout should contain "just to fail and print the result to console:"
```

Comment:
Why this scenario is repeated at diffferent place in this file?

Reply:
@praveenkumar just to get information about docker-registry container. It seems that after restart of the OpenShift the container is not running: 
```
08:30:56                 "containerStatuses": [
08:30:56                     {
08:30:56                         "containerID": "docker://de3d6280c9d576441cca4dcadd897d3b17a0a71bae280507a23807c94635004a",
08:30:56                         "image": "openshift/origin-deployer:v3.7.1",
08:30:56                         "imageID": "docker-pullable://openshift/origin-deployer@sha256:064c3ba908fe9f7b4a22373bde98167789292de3670d48b05ef665748d1cadea",
08:30:56                         "lastState": {},
08:30:56                         "name": "deployment",
08:30:56                         "ready": false,
08:30:56                         "restartCount": 0,
08:30:56                         "state": {
08:30:56                             "terminated": {
08:30:56                                 "containerID": "docker://de3d6280c9d576441cca4dcadd897d3b17a0a71bae280507a23807c94635004a",
08:30:56                                 "exitCode": 1,
08:30:56                                 "finishedAt": "2018-03-14T08:30:33Z",
08:30:56                                 "reason": "Error",
08:30:56                                 "startedAt": "2018-03-14T08:30:16Z"
08:30:56                             }
08:30:56                         }
08:30:56                     }
```